grr/server/grr_response_server/gui/api_plugins/metadata.py

```diff
@@ -132,6 +133,66 @@ class RouteInfo(NamedTuple):
   },
 }
 
+rdf_types_schemas: Dict[str, Schema] = {
```

Comment:
I am not a native speaker but double pluralization sounds awkward to me. Shouldn't it be `rdf_type_schemas`?

Reply:
I am not a native speaker either, but the way I thought about it when I named the variable was that the dictionary will hold different `schemas` for different `rdf_types`, whereas `rdf_type_schemas` sounds to me like a structure for multiple `schemas` associated with the same, single `rdf_type`. Please let me know if I should change it.

Reply:
Maybe @bgalehouse can provide some expertise here.

Reply:
I think you want rdf_type_schemas.

It is a semi-corner case in english that nouns used as adjectives lose any plurality marker, even if the noun would otherwise be trivially plural.  "5 second rule", "2 foot drop", _not_ "5 seconds rule" or "2 feet drop".

Reply:
Thank you @panhania and @bgalehouse ! There are multiple instances of this wrong naming scheme, even for parts of code outside of this PR so I think I should create a separate PR fixing all occurrences. Please let me know if I should change this specific one in this PR.

Reply:
I think it is fine to do it in a separate PR.

Reply:
BTW, my suggestion gives the impression that you a number of schemas, each of which is connected/related to one or more rdf_type values - each entry could be called an rdf_type schema.

If the schemas and the rdf_types just happen to be attached, something like rdf_type_to_schema could work as a name for a dict/map.